modules/chakra-ui.yml

```diff
@@ -0,0 +1,14 @@
+name: chakra-ui
+description: !<tag:yaml.org,2002:js/undefined> ''
```

Comment:
Description is wrong @Geminii

Reply:
We got this error into another sync module probably due to the description package. 
I will be careful for next integration